lib/document/stateTransition/applyDocumentsBatchTransition.js

```diff
@@ -0,0 +1,44 @@
+const applyDocumentTransition = require('./documentTransition/applyDocumentTransition');
+
+/**
+ * Select document by id among documents array
+ *
+ * @param {string} id
+ * @param {Document[]} documents
+ *
+ * @return {Document|null}
+ */
+function findDocumentById(id, documents) {
+  const [foundDocument] = documents
+    .filter((document) => document.getId() === id);
+
+  return foundDocument;
+}
+
+/**
+ * Apply documents batch transition
+ *
+ * @param {DocumentsBatchTransition} stateTransition
+ * @param {Document[]} documents
+ *
+ * @returns { { id: string, document: Document|null }|{} }
+ */
+function applyDocumentsBatchTransition(stateTransition, documents) {
```

Comment:
May be input and output should have the same format? I mean it should accept a map too?